src/Sarif/CodeGenHints.json

```diff
@@ -379,6 +521,14 @@
   "Run.OriginalUriBaseIds": [
     {
       "kind": "DictionaryHint"
+    },
+    {
+      "kind": "AttributeHint",
+      "arguments": {
+        "namespaceName": "Newtonsoft.Json",
+        "typeName": "JsonConverter",
```

Comment:
>JsonConverter [](start = 21, length = 13)

Same question: Does the converter work on the object's property values when you apply the attribute to an object?

Reply:
Ok, I just found the code you wrote to dive into the dictionary.

---
In reply to: [221316364](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221316364) [](ancestors = 221316364)